starter-code/functions-and-arrays.js

```diff
@@ -64,6 +87,15 @@ console.log(averageLength);
 // Unique Arrays
 function uniquifyArray (array) {
 
+  var uniqueArr = [];
+
+  array.forEach(function(word) {
+    if (uniqueArr.includes(word)) {
```

Comment:
This function is returning an empty array because uniqueArr don't include anything at the beginning. Try this:
```
function uniquifyArray (array) {
  var uniqueArr = [];
  array.forEach(function(word) {
    if (!uniqueArr.includes(word)) {
      uniqueArr.push(word);
    }
  });
  return uniqueArr;
}
```
You can read this "if (!uniqueArr.includes(word))" as "if uniqueArr DON'T include the word, then...".